p-defer/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
Good question. I created a Yeoman generator ([generator-dt](https://github.com/SamVerschueren/generator-dt)) that scaffolds out most of the files. It looks like I picked a `tslint` without strict null checks as the base of my generator. I'll remove it.